### PR TITLE
Update Volume Plasmon

### PR DESCRIPTION
**Pull request by erh3cq**
Gave the dampening energy a more descriptive name.
Also, allowed for introduction of initial parameters during component
creation.

Also, Let me know if I am doing this whole push/pull thing correctly

### CODE REVIEW

**Comment by francisco-dlp** [comment]
Looks good, except for changing the damping energy name: changing names breaks people's code and it is only allowed in major releases, so we only do it when strictly necessary.

**Comment by erh3cq** [comment]
Alright, help me out here.  I changed it back to fwhm. Now how do I re-submit to the same pull? Then open a new pull for a different file I changed?

**Comment by francisco-dlp** [comment]
Just ``git push`` will update this PR.

File: hyperspy/_components/volume_plasmon_drude.py [hunk 1]
```diff
@@ -50,13 +50,13 @@ class VolumePlasmonDrude(Component):
 
     """
 
-    def __init__(self):
+    def __init__(self, intensity = 1., plasmon_energy = 15., fwhm = 1.5):
```
Comment (francisco-dlp) [inline hyperspy/_components/volume_plasmon_drude.py]: No spaces around ``=``, see [PEP8](https://www.python.org/dev/peps/pep-0008/)

**Comment by erh3cq** [comment]
Sweet done.  Now how do I open a completely new pull request?  I want to send a new function, but it looks as if it will post to this pull

**Comment by francisco-dlp** [comment]
You have to branch from RELEASE_next_minor. If you didn't add the hyperspy repository to your remotes:

```bash
git remote add upstream https://github.com/hyperspy/hyperspy.git
```

Then 

```bash

git fetch upstream
git checkout upstream/RELEASE_next_minor
git checkout -b your_new_branch

```

To push to your repository (assuming the remote is ``origin``):

```bash

git push --set-upstream origin your_new_branch

```

File: hyperspy/_components/volume_plasmon_drude.py [hunk 2]
```diff
@@ -50,13 +50,13 @@ class VolumePlasmonDrude(Component):
 
     """
 
-    def __init__(self):
+    def __init__(self, intensity = 1., plasmon_energy=15., fwhm=1.5):
```
Comment (francisco-dlp) [inline hyperspy/_components/volume_plasmon_drude.py]: I can still find spaces there...

**Comment by erh3cq** [comment]
Yeah I already created a branch.  Do I have to make a second branch?

**Comment by francisco-dlp** [comment]
Thanks, waiting for travis before merging.

**Comment by francisco-dlp** [comment]
It depends on where you have branched it from. If you have branched from RELEASE_next_minor it's fine, otherwise you should create a new branch branching from Rnm.

**Comment by erh3cq** [comment]
I did branch from there.  See just did another commit, now my eels update is in the pull request. Figured you would like them separate.

**Comment by francisco-dlp** [comment]
You are committing to the wrong branch. Could you revert your last commit?

```bash

git revert 3ae6565

```

and push again?

**Comment by francisco-dlp** [comment]
I am afraid that you've commited to the same branch again. Could you revert the last commit? Afterwards you can follow the instructions above to create a new branch to commit the new changes.

**Comment by erh3cq** [comment]
Finally!. This is very different to me...

Thank you Francisco!

**Comment by francisco-dlp** [comment]
No worries, we've all struggled with these things when we started.

You still  have to revert the last commit:

```bash
git revert 644da045e7b988
git push

```

**Comment by erh3cq** [comment]
Finally got the correct files to the correct pulls!  I am in need of some serious help with this whole Git system.  I have other files that I would like to add, but my lack of understanding is making this frustrating.  I use GitHub and GitHub desktop.  So it automatically handles the upstream for me (I think).  When I would like to create a new pull, I should create a new branch off of RELEASE_next_minor (which I forked from hyperspy)? Then Commit and sync(push) to the new branch?

@francisco-dlp thank you for your patience and help

**Comment by francisco-dlp** [comment]
Yes, that should be it. I don't have any experience with GH desktop, so I can't help much there.

The deconvolution function is till there and the tests are not passing. I would suggest to send a new PR with these changes. First update your RELEASE_next_minor from upstream, then branch from it and finally commit your changes and push to your repository. There is some more help with GH [here](http://hyperspy.readthedocs.io/en/latest/dev_guide.html).

**Pull request closed without merging**